tests/cypress/e2e/loan/peg-stability-reserves.ts

```diff
@@ -15,10 +16,9 @@ describe(`Peg Stability Reserves`, () => {
     cy.get('[data-testid="pegkeepers"]', LOAD_TIMEOUT).should('be.visible')
   })
 
-  // Ideally I would match it with the number of entries from PEG_KEEPERS, but that import from @/loan isn't available in tests
   it('should render all pegkeepers', () => {
-    cy.get('[data-testid^="pegkeeper-"]').should('have.length.greaterThan', 0)
-    cy.get('[data-testid^="pegkeeper-"]').each(($el) => {
+    cy.get('[data-testid^="pegkeeper-card"]').should('have.length', PEG_KEEPERS.length)
```

Comment:
The tests are failing here
<img width="721" height="504" alt="image" src="https://github.com/user-attachments/assets/c71c7d6d-8a61-400b-be85-b4d591ebf79e" />

Reply:
Should be fixed now, I changed some testids for the other tests